src/protocol/codecs/msgpack.rs

```diff
@@ -382,6 +462,181 @@ impl<'de> Deserialize<'de> for HashDigest {
     }
 }
 
+/// An Ed25519 Signature.
+#[derive(Copy, Clone, Eq)]
+pub struct Ed25519Signature(pub [u8; 64]);
```

Comment:
64 could be some const to avoid magic numbers.

Reply:
From the internal discussion:
we'll keep using raw numbers (no additional consts) in struct definitions where the length is not referenced anywhere in the code logic. (because messing up the number in definitions will cause compiler warnings, so we are safe - and code doesn't get cluttered with too many constants for everything)